internal/signing/google_cloud_kms.go

```diff
@@ -40,9 +40,5 @@ func NewGoogleCloudKMS(ctx context.Context) (KeyManager, error) {
 }
 
 func (kms *GoogleCloudKMS) NewSigner(ctx context.Context, keyID string) (crypto.Signer, error) {
-	signer, err := gcpkms.NewSigner(ctx, kms.client, keyID)
-	if err != nil {
-		return nil, err
-	}
-	return signer, nil
+	return gcpkms.NewSigner(ctx, kms.client, keyID)
```

Comment:
wow - don't know why I did this before. duh.

Reply:
You were buffing your LOC stats 😛